src/content/docs/ra-primer/ra-in-action.mdx

```diff
@@ -94,10 +94,10 @@ In this case, we want rows where the **name** attribute from `sk_films` equals t
 sk_roles = roles.join(sk_films, :film_name => :name)
 ```
 
-| film_name | production_year | director | actor_name | name |
```

Comment:
These are the actors' names. I think you misunderstood. (Probably confusing because Dr. Strangelove is the name both of the film and a character. Can be improved in the future by choosing another film.)